src/esp/sim/AbstractReplayRenderer.cpp

```diff
@@ -91,12 +91,23 @@ void AbstractReplayRenderer::setSensorTransformsFromKeyframe(
 }
 
 void AbstractReplayRenderer::render(
-    Cr::Containers::ArrayView<const Mn::MutableImageView2D> imageViews) {
-  CORRADE_ASSERT(imageViews.size() == doEnvironmentCount(),
-                 "ReplayRenderer::render(): expected" << doEnvironmentCount()
-                                                      << "image views but got"
-                                                      << imageViews.size(), );
-  return doRender(imageViews);
+    Corrade::Containers::ArrayView<const Magnum::MutableImageView2D>
```

Comment:
```suggestion
    Cr::Containers::ArrayView<const Mn::MutableImageView2D>
```

so it's not so excessively long. Same elsewhere and in other `*.cpp` files -- but not headers, that breaks cross-linking / confuses the generated documentation.

Given it was using the shorthands before, I suspect an IDE is to blame for expanding this? :D I don't have a solution for IDEs picking the shorter variant, unfortunately.